api/top-langs.js

```diff
@@ -77,6 +78,7 @@ export default async (req, res) => {
         border_color,
         locale: locale ? locale.toLowerCase() : null,
         disable_animations: parseBoolean(disable_animations),
+        hide_progress: parseBoolean(hide_progress) ? true : false,
```

Comment:
Why is this needed? Simply `hide_progress: parseBoolean(hide_progress)` should work.

Reply:
Yes, it's true. I just wanted to prevent undefined values. And somehow explicitly say the default value for it.

Reply:
We already take care of this:
https://github.com/anuraghazra/github-readme-stats/blob/1df150127b6e130195cf0145ebb432a2307e6255/src/common/utils.js#L83-L94

Please remove this line to keep the code nice and clean (it's not needed since we're already accounting for undefined values). Thank you!

Reply:
Done. Thank you